languages/ruby/lib/oso/polar/query.rb

```diff
@@ -152,6 +152,8 @@ def handle_make_external(data) # rubocop:disable Metrics/AbcSize
         host.make_instance(cls_name, args: args, kwargs: kwargs, id: id)
       end
 
+      public
+
```

Comment:
Why does `Query#each` have to be public? If it does have to, I'd prefer to move it up with the other public method (`Query#initialize`) instead of having this random public block midway through the private section of the class

Reply:
It's kinda weird that you can't enumerate results using `.each` given that `Query` is an enumerable (the `.each` happens in `authorized_{actions,fields}`). We can call `.to_a.each`, but that feels weird.

I'll move this up to the public section -- will be big LOC change.

Reply:
I actually feel like `.each` should be public but there should be a kind of internal private `.run` method that's called by `.each`.

Just made that change -- what do you think?

Reply:
Don't feel strongly about it; :shipit: